setup.py

```diff
@@ -11,7 +11,7 @@
     name="mdf_matio",
     version=version,
     packages=find_packages(),
-    install_requires=['pypif_sdk', 'jmespath>=0.9.4', 'jsonschema>3'],
+    install_requires=['pypif_sdk', 'jmespath>=0.9.4', 'jsonschema>3', 'mdf_toolbox>=0.5.2'],
     include_package_data=True,
     entry_points={
         'materialsio.adapter': [
```

Comment:
Eventually, we need to add in entry_points for the adapters so that MaterialsIO can find them.